### PR TITLE
Further refactors

### PR DESCRIPTION
**Pull request by C-Loftus**
I cleaned up a lot of old code, I have a better understanding of the dagster ops file now in general.

Everything is still passing 
image

### CODE REVIEW

File: templates/implnet_ops_SOURCEVAL.py [hunk 1]
```diff
@@ -119,82 +141,46 @@ def strict_env(key: str):
...
+    return f"{GLEANER_GRAPH_URL}/namespace/{GLEANER_GRAPH_NAMESPACE}_summary/sparql"
 
 def _pythonMinioAddress(url, port=None) -> str:
     """Construct a string for connecting to a minio S3 server"""
     if not url:
         raise RuntimeError("Tried to construct minio address with an empty URL")
-    get_dagster_logger().info(f"{url=}{port=}")
-
-    if url.endswith(".amazonaws.com"):
-        PYTHON_MINIO_URL = "s3.amazonaws.com"
```
Comment (C-Loftus) [inline templates/implnet_ops_SOURCEVAL.py:138]: Leaving it as client responsibility to give the right url. I dont think we want to special case check all these s3 urls

File: templates/implnet_ops_SOURCEVAL.py [hunk 2]
```diff
@@ -293,177 +260,145 @@ def _create_service(
         networks=container_context.networks
         if len(container_context.networks)
         else None,
-        restart_policy=restart_policy,
-        mode=serivce_mode,
+        restart_policy= RestartPolicy(condition="none"),
+        mode=ServiceMode("replicated-job", concurrency=1, replicas=1),
         workdir=workingdir,
-        configs=configs,
```
Comment (C-Loftus) [inline templates/implnet_ops_SOURCEVAL.py:299]: Don't like the pattern of defining a variable just to use it in func arg if it is very minor

File: templates/implnet_ops_SOURCEVAL.py [hunk 3]
```diff
@@ -293,177 +260,145 @@ def _create_service(
...
         NAME = f"sch_{source}_{str(mode)}"
-        WorkingDir = "/nabu/"
-        Entrypoint = "nabu"
-        # LOGFILE = 'log_nabu.txt'  # only used for local log file writing
-    else:
-        returnCode = 1
-        return returnCode
+        match mode:
+            case "release":
+                ARGS = [
```
Comment (C-Loftus) [inline templates/implnet_ops_SOURCEVAL.py:304]: Match is cleaner and gives us intellisense. Some duplicate code refactored out

File: templates/implnet_ops_SOURCEVAL.py [hunk 4]
```diff
@@ -474,62 +409,26 @@ def gleanerio(context, mode, source):
...
-        environment_variables = [
-            f"MINIO_ADDRESS={GLEANER_MINIO_ADDRESS}",
-            f"MINIO_PORT={GLEANER_MINIO_PORT}",
-            f"MINIO_USE_SSL={GLEANER_MINIO_USE_SSL}",
-            f"MINIO_SECRET_KEY={GLEANER_MINIO_SECRET_KEY}",
-            f"MINIO_ACCESS_KEY={GLEANER_MINIO_ACCESS_KEY}",
-            f"MINIO_BUCKET={GLEANER_MINIO_BUCKET}",
-            f"SPARQL_ENDPOINT={_graphEndpoint()}",
-            f"GLEANER_HEADLESS_ENDPOINT={GLEANER_HEADLESS_ENDPOINT}",
-            f"GLEANER_HEADLESS_NETWORK={GLEANER_HEADLESS_NETWORK}",
```
Comment (C-Loftus) [inline templates/implnet_ops_SOURCEVAL.py:493]: some of the env stuff for dagster is still a bit opaque but it seems that this is unnecessary and it runs fine without it as long as we run `container_context = run_container_context.merge(op_container_context)`

**Review by webb-ben — APPROVED** [review]

**Pull request merged**